src/solve.jl

```diff
@@ -281,6 +282,7 @@ function DiffEqBase.__init(prob::DiffEqBase.AbstractDDEProblem,
                                                                                progress_steps,
                                                                                progress_name,
                                                                                progress_message,
+                                                                               progress_id,
```

Comment:
this will need to `isdefined(fieldnames(OrdinaryDiffEq.DEOptions), :progress_id)` for backwards compatability.

Reply:
Is there an elegant way to just omit the argument, or you have to copy the whole instantiation?

Reply:
I think the easiest is just to copy. It can be removed in the future.

Reply:
alright